target/linux/ath79/image/generic.mk

```diff
@@ -49,6 +49,15 @@ define Device/glinet_ar300m_nor
 endef
 TARGET_DEVICES += glinet_ar300m_nor
 
+define Device/ocedo_raccoon
+  ATH_SOC := ar9344
+  DEVICE_TITLE := OCEDO Raccoon
+  IMAGE_SIZE := 7424k
+  SUPPORTED_DEVICES += raccoon
```

Comment:
Please add also the full compatible string "ocedo,raccoon"

Reply:
No need to add "ocedo,raccoon".

`+=` extends the list of supported devices which is defined in [Device/default](/target/linux/ath79/image/Makefile) as `SUPPORTED_DEVICES := $(subst _,$(comma),$(1))`. `$1` is the device name here and the result will be `SUPPORTED_DEVICES := ocedo,raccoon`.